Packages/ClientRuntime/Sources/Networking/Http/UnknownHttpServiceError.swift

```diff
@@ -19,9 +21,24 @@ public struct UnknownHttpServiceError: HttpServiceError, Swift.Equatable {
 }
 
 extension UnknownHttpServiceError {
-    public init(httpResponse: HttpResponse, message: String? = nil) {
+
+
+    /// Creates an `UnknownHttpServiceError` from a HTTP response.
+    /// - Parameters:
+    ///   - httpResponse: The `HttpResponse` for this error.
+    ///   - message: The message associated with this error.  May be omitted.
+    ///   - errorCode: The error code associated with this error.  May be omitted.
```

Comment:
Typically docs will say what the value defaults to.
```suggestion
    ///   - message: The message associated with this error. Defaults to nil
    ///   - errorCode: The error code associated with this error.  Defaults to nil
```